### ISSUE TITLE
zoomFactor is not applied

### ISSUE THREAD

**Issue by javismiles** [laurentj/slimerjs#75]
Hi, some issues while testing slimerjs on centos 6 linux:

1) the page https://github.com/laurentj/slimerjs/blob/master/API_COMPAT.md
says that zoomFactor is implemented but i tried it and doesn't seem to work. Scaling doesn't happen. Is the use of zoomFactor different than in PhantomJS?

Im currently using this code to test:
<?php
$command="xvfb-run /var/www/vhosts/(rest of path)slimerjs/slimerjs slimtest.js";
exec($command);
?>

and my script is
var page = require('webpage').create();
page.open("http://volandino.com/fonts/fonts.html", function (status) {
//    page.viewportSize = { width:1920, height:1080 };
    page.viewportSize = { width:9600, height:5400 };
    page.zoomFactor = 5;  
    page.render('screenshot2.png')
});

2) I launched a screen with this command
Xvfb :99 -ac -screen 0 1024x768x16

and then When executing slimerJS it always Hangs, something with the xvfb server maybe,  if i do it from php it doesnt work or produce anything, it just hangs, if i execute it from Command line Shell then it generates the PNG image well but it also hangs and i have to stop it with CTRL-C , do you know how to make it work without Hanging and also how to make it work from php ?

it only generates image from command line shell, and it hangs both from command line and from php 

3) 
finally one more thing, the render of webFonts is perfect on png, very happy,
but there is slight color fringing in some letters and also edges of letters are not as smooth as in phantomjs i feel, any opinions about that? Howeves shapes of webfonts come out perfect (unlike in phantomjs) which is so so so important, very happy with that

thank you very much for your help
Jav

**Comment by puravida** [comment]
This zoom feature is just a bonus to me, so no pressure, but was this ever confirmed as fixed? I just tried it with v0.10.0 and it doesn't seem to work with a 20 second timeout at 1024x768 and zoom of 2 or 5. I read the docs and also set a slimer.wait(500) after setting the zoomFactor, but it made no difference. The image always looked the same in all tests.

**Comment by puravida** [comment]
Just a quick update: I took a few minutes to revisit this before writing it off. I am using an 8Gb, 2 vCPU EC2 VM for testing this currently. This is the only thing running and I set the delay to 30 seconds and no value for zoomFactor seems to work. After re-reading this thread, I thought it might be a timing issue, still, so I moved the page.zoomFactor update inside the page.open() but outside of the page.render() w/ setTimeout(). Still no go. Just updating for posterity, since I'm unable to use this feature.
